.circleci/RevealAuthoringKeys.js

```diff
@@ -0,0 +1,5 @@
+console.log(`process.env.LUIS_AUTHORING_KEY_ALT_1=${process.env.LUIS_AUTHORING_KEY_ALT_1}\n`)
```

Comment:
The `SetLuisAuthoringKey.js` was already risky but accepted because output was piped to the $BASH_ENV

This `- run: node .circleci/RevealAuthoringKeys.js` is run directly which I think means it output is shown in the circle CI logs. We can't expose the environment variables to those logs I think these are also public.

Reply:
You will have to rotate all these keys now

Reply:
When you say rotate it sounds like I just need to re-order the keys in the array in SetLuisAuthoringKey.js file. I am okay with doing it that way, but I would assume I need to get whole new keys and delete the old ones...is that what you really meant?

Reply:
Yes, should have said regenerate all of them. rotate is for difference process when keys expire and you have to regenerate one of them.

> I would assume I need to get whole new keys and delete the old ones

Yes, they are leaked to public now.  Dont expose secrets